.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,15 @@
 <!--
-Thank you for reporting an issue! Please review the following notes before submitting it.
+Thank you for reporting an issue!
+Please review the following notes before submission.
 
-Most issues on this repo are requests for new commands. If it's the case, make sure to use the standard title ("page request: <command name>"), and include a link to a web page about the command, and if possible, to an existing source of example-style documentation.
+If you request a new command, please:
+- use the standard title ("page request: <command name>")
+- include a link to a web page about the command
+- if available, add an existing source of examples
```

Comment:
Perhaps this will read better?
```suggestion
- if available, add an existing source of example-style documentation
```

Reply:
I personally have no idea what "example-style documentation" is

I thought it meant: Add a link to a place that already lists examples, i.e. a "source of examples"

Reply:
@xeruf I think `example-style documentation` means something like `documentation build up of examples`. I like the phrasing, but I think we should stick to more simple English.

Reply:
Personally I prefer `example-style documentation`, because it emphasizes that it should be proper documentation and not just any kind of examples, but `examples` is definitively easier for non-English speakers and good enough IMO.